src/lib/holocene/modal.svelte

```diff
@@ -22,7 +22,10 @@
         class="float-right cursor-pointer p-6"
```

Comment:
Unrelated, but it might be nice to set `cursor-default` on the root `div` so it's not a pointer for the entire page/modal.

Reply:
Good call, added cursor-default on the top level div